nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/ElasticSearchComponent.java

```diff
@@ -272,15 +272,19 @@ public String getIndexNameForType(String type) {
         return esa.getIndexNameForType(type);
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Why removing the deprecation message? This will have to be fixed at some time. `org.nuxeo.ecm.core.work.api.WorkManager.getQueueSize(String, State)` is deprecated since 5.8